app/Console/Commands/InstallCommand.php

```diff
@@ -109,9 +111,9 @@ protected function configureDatabase(array $default = [])
         ], $default);
 
         $config['DB_DRIVER'] = $this->choice('Which database driver do you want to use?', [
-            'mysql'      => 'MySQL',
-            'postgresql' => 'PostgreSQL',
-            'sqlite'     => 'SQLite',
+            'mysql'  => 'MySQL',
+            'pgsql'  => 'PostgreSQL',
```

Comment:
I guess the most correct way to do it would be to filter laravel's returned results by a the 3 we do support, rather than removing the one we know we don't.